bolt/lib/Rewrite/LinuxKernelRewriter.cpp

```diff
@@ -313,6 +310,9 @@ class LinuxKernelRewriter final : public MetadataRewriter {
     if (Error E = readAltInstructions())
       return E;
 
+    if (Error E = readORCTables())
+      return E;
```

Comment:
we might want to add a comment here explaining why there's this order, i.e. we first have to read `.parainstruction` and `.altrinstruction` as they're a dependency for ORC